File: uhrwerk-common/src/main/java/io/qimia/uhrwerk/common/metastore/config/TargetResult.java
package io.qimia.uhrwerk.common.metastore.config;

import io.qimia.uhrwerk.common.model.TargetModel;

import java.io.Serializable;

public class TargetResult implements Serializable {

    private static final long serialVersionUID = 740323809677539485L;
    TargetModel[] storedTargets;
    boolean success;
    boolean error;
    String message;
    Exception exception;

    public TargetModel[] getStoredTargets() {
        return storedTargets;
    }

    public void setStoredTargets(TargetModel[] storedTargets) {
        this.storedTargets = storedTargets;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public boolean isError() {
        return error;
    }

    public void setError(boolean error) {
        this.error = error;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Exception getException() {
        return exception;
    }

    public void setException(Exception exception) {
        this.exception = exception;
    }
}

File: uhrwerk-metastore/src/main/kotlin/io/qimia/uhrwerk/common/metastore/config/TargetResult.kt
package io.qimia.uhrwerk.common.metastore.config

import io.qimia.uhrwerk.common.model.TargetModel
import java.io.Serializable

class TargetResult : Serializable {
    var storedTargets: Array<TargetModel>? = null
    var isSuccess = false
    var isError = false
    var message: String? = null
    var exception: Exception? = null
    override fun toString(): String {
        return ("TargetResult{"
                + "success="
                + isSuccess
                + ", error="
                + isError
                + ", message='"
                + message
                + '\''
                + ", exception="
                + exception
                + '}')
    }

    companion object {
        private const val serialVersionUID = 740323809677539485L
    }
}